apps/educational_institutions/schemas.py

```diff
@@ -45,7 +45,7 @@ class FacultyOut(BaseOutSchema):
     shortname: str
     main_email: str = None
     university_id: int
-    dekan_full_name: str = None
+    dean_full_name: Dict[str, str] = None
```

Comment:
Should be an embedded schema, that describes this structure

Reply:
example?

Reply:
fixed